src/modules/main/components/MainExpertsView.tsx

```diff
@@ -30,26 +37,28 @@ export const MainExpertsView: React.FC = () => {
   const allExperts = cards.map((card, key) => (
     <>
       <div
-        onMouseEnter={handlePopoverOpen}
+        onMouseEnter={(event) => handlePopoverOpen(event, card)}
```

Comment:
you can search the possibility for getting the expert or card from the event itselft, so that it'd be possible to reduce the number of arguments.

but, anyway, what it is now, it's ok.

Reply:
I searched but couldn't find a way to receive the card from the event itself.